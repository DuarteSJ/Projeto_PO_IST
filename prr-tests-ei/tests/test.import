CLIENT|1|Manuel Pinheiro|103443
CLIENT|2|Pedro Pinheiro|103447
CLIENT|3|Jo√£o Pinheiro|103125
CLIENT|4|John Pattenden|102546
FANCY|111111|1|ON
FANCY|222222|2|ON
FANCY|333333|3|ON
FANCY|444444|4|OFF
