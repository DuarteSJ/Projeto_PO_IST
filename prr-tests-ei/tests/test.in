4
3
111111
9
444444
VOICE
8
444444
olá
0
3
222222
9
444444
VOICE
0
3
333333
8
444444
olá
0
3
444444
2
1
0
0
3
1
1
1
2
1
3
0
0
