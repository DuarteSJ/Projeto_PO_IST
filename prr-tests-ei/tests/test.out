MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): O terminal de destino ('444444') está desligado.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: O terminal de destino ('444444') está desligado.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Tipo de comunicação a iniciar (VOICE, VIDEO): O terminal de destino ('444444') está desligado.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: Número do terminal: Mensagem de texto a enviar: O terminal de destino ('444444') está desligado.
OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: OPERAÇÕES DO TERMINAL
1 - Ligar
2 - Colocar no silêncio
3 - Desligar
4 - Adicionar amigo
5 - Retirar amigo
6 - Pagamento
7 - Consultar saldo
8 - Enviar comunicação de texto
9 - Iniciar comunicação
11 - Mostrar comunicação em curso
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: Identificador do cliente: CLIENT|1|Manuel Pinheiro|103443|NORMAL|YES|1|0|0
O2S|444444
S2I|444444
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: Identificador do cliente: CLIENT|2|Pedro Pinheiro|103447|NORMAL|YES|1|0|0
S2I|444444
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: Identificador do cliente: CLIENT|3|João Pinheiro|103125|NORMAL|YES|1|0|0
O2S|444444
GESTÃO DE CLIENTES
1 - Visualizar cliente
2 - Visualizar todos os clientes
3 - Registar cliente
4 - Activar notificações
5 - Desactivar notificações
6 - Mostrar saldo de cliente
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 